src/main/java/wooteco/subway/exception/ExceptionAdvice.java

```diff
@@ -0,0 +1,34 @@
+package wooteco.subway.exception;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestControllerAdvice
+public class ExceptionAdvice {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @ExceptionHandler(SubwayException.class)
+    public ResponseEntity<Object> handleSubwayException(SubwayException e) {
+        logger.error(String.valueOf(e.getBody()));
+        return ResponseEntity.status(e.getHttpStatus()).body(e.getBody());
+    }
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<Map<String, String>> handleValidationException(MethodArgumentNotValidException e) {
+        Map<String, String> errors = new HashMap<>();
+        e.getBindingResult().getAllErrors()
+                .forEach(error -> {
+                    errors.put(((FieldError) error).getField(), error.getDefaultMessage());
+                    logger.error("%s : %s", ((FieldError) error).getField(), error.getDefaultMessage());
```

Comment:
logger 를 사용하면 아래처럼 사용 할 수 있어요 :) 
```suggestion
                    logger.error("{} : {}", ((FieldError) error).getField(), error.getDefaultMessage());
```